proto/Networking/Requests/Messages/LocationPingMessage.proto

```diff
@@ -1,10 +1,7 @@
 syntax = "proto3";
 package POGOProtos.Networking.Requests.Messages;
 
-message UpdatePlayerLocationMessage {
```

Comment:
request type is `UPDATE_PLAYER_LOCATION` so that was correct

Reply:
https://github.com/Furtif/POGOProtos/blob/master/src/POGOProtos/Enums/PlatformClientAction.proto#L39

Reply:
same as before, that's not our naming convention

Reply:
fixed https://github.com/pogosandbox/node-pogo-protos/pull/19/commits/2699be05eaf2516680c0121818fc42aa5a287daf